lib/pf/CHI.pm

```diff
@@ -1,3 +1,11 @@
+package pf::Redis;
```

Comment:
That I saw :)

I mean since you have pf::Redis in lib/pf/Redis.pm and you redefine it in another module which is not in our standards